assets/js/event_admin.js

```diff
@@ -0,0 +1,128 @@
+    currentSeats = currentSeats.filter(seat => seat[0] !== gridX || seat[1] !== gridY);
+  }
+}
+
+window.onload = function () {
+
+  // set up form submission hook
+  var form = document.getElementById("event_form");
+  var canvasInput = document.createElement("input");
+  canvasInput.setAttribute("name", "seats");
+  canvasInput.setAttribute("type", "hidden");
+  canvasInput.setAttribute("value", "");
+  form.appendChild(canvasInput);
+
+  form.addEventListener("submit", event => {
+    pixels = JSON.stringify(currentSeats);
+    canvasInput.setAttribute("value", pixels);
+  });
+
+  var canvas = document.getElementById("seatCanvas");
+  var ctx = canvas.getContext("2d");
+
+  // get old seats
+  let oldSeats = document.getElementById("id_oldseats").value;
+  JSON.parse(oldSeats).forEach(seat => {
+    currentSeats.push(seat)
+    placePixel(ctx, seat[0], seat[1], pickedColor);
+  });
+
+  let data = {
+    lastHoveredCell: null,
+    oldState: null,
+  }
+
+  drawGrid();
+
+  canvas.addEventListener('mousemove', event => {
+    const bounding = canvas.getBoundingClientRect();
+    const x = event.clientX - bounding.left;
+    const y = event.clientY - bounding.top;
+
+    // calculate the grid cell coordinates
+    const gridX = Math.floor(x / gridSize);
+    const gridY = Math.floor(y / gridSize);
+
+    let lastHoveredCell = data.lastHoveredCell;
+
+    // if the mouse has moved to a new cell, redraw the grid
+    if (lastHoveredCell === null || gridX !== lastHoveredCell.x || gridY !== lastHoveredCell.y) {
+      // restore the canvas to its state before the outline was drawn
+      if (data.oldState !== null) {
+        ctx.putImageData(data.oldState, 0, 0);
+      }
+
+      // save the current state of the canvas
+      const imageData = ctx.getImageData(0, 0, canvas.width, canvas.height);
+
+      // draw the outline on the current cell
+      ctx.strokeStyle = 'red'; // color of the outline
+      ctx.lineWidth = 2; // width of the outline
+      ctx.strokeRect(gridX * gridSize, gridY * gridSize, gridSize, gridSize);
+
+      // store the current cell and the image data
+      data.lastHoveredCell = { x: gridX, y: gridY };
+      data.oldState = imageData
+    }
+  });
+
+
+  canvas.addEventListener('click', event => {
+    const bounding = canvas.getBoundingClientRect();
+    const x = event.clientX - bounding.left;
+    const y = event.clientY - bounding.top;
+
+    // calculate the grid cell coordinates
+    const gridX = Math.floor(x / gridSize);
+    const gridY = Math.floor(y / gridSize);
+
+    ctx.putImageData(data.oldState, 0, 0);
+
+    // javascript is cursed
+    let picked = currentSeats.some(seat => seat[0] === gridX && seat[1] === gridY);
+    let color = picked ? notPickedColor : pickedColor;
+
+    placePixel(ctx, gridX, gridY, color);
+
+    // save the current state of the canvas
+    const imageData = ctx.getImageData(0, 0, canvas.width, canvas.height);
+    data.oldState = imageData
+  });
+}
```

Comment:
It would be really nice if you add the possibility to drag pixels to chose several near.